lib/analytics/index.js

```diff
@@ -0,0 +1,22 @@
+import Analytics from "analytics";
+import segmentPlugin from "@analytics/segment";
+import googleAnalytics from "@analytics/google-analytics";
+
+const analytics = Analytics({
+  app: "example-storefront",
+  plugins: [
+    segmentPlugin({
+      writeKey: process.env.SEGMENT_ANALYTICS_WRITE_KEY
+    }),
+    googleAnalytics({
+      trackingId: process.env.GOOGLE_ANALYTICS_ID
+    })
+  ]
+});
+
+/** Disable plugins that it's not configured properly */
+if (process.env.GOOGLE_ANALYTICS_ID === "") analytics.plugins.disable(["google-analytics"]);
```

Comment:
I'm wondering, instead of a separate call to disable plugins in a separate call after loading them, couldn't we just conditionally enable them based on the env value right at the definition of the `plugins` array?